float32[6] current #amps
float32[6] accel #deg/s^2
float32[6] vel #deg/s
float32[6] angle #deg
uint32 msg_ctr # count sent msgs to detected missed messages